optimum/onnxruntime/modeling_ort.py

```diff
@@ -171,16 +172,25 @@ def load_model(
             provider (`str`, *optional*):
                 ONNX Runtime provider to use for loading the model. See https://onnxruntime.ai/docs/execution-providers/
                 for possible providers. Defaults to `CPUExecutionProvider`.
-            session_options (`onnxruntime.SessionOptions`, *optional*),:
+            session_options (`onnxruntime.SessionOptions`, *optional*):
                 ONNX Runtime session options to use for loading the model. Defaults to `None`.
+            provider_options (`Dict`, **optional**):
+                Provider option dictionaries corresponding to the provider used. See available options
+                for each provider: https://onnxruntime.ai/docs/api/c/group___global.html . Defaults to `None`.
         """
         available_providers = ort.get_available_providers()
         if provider not in available_providers:
             raise ValueError(
                 f"Asked to use {provider} as an ONNX Runtime execution provider, but the available execution providers are {available_providers}."
             )
 
-        return ort.InferenceSession(path, providers=[provider], sess_options=session_options)
+        # `providers` list must of be of the same length as `provider_options` list
+        return ort.InferenceSession(
+            path,
+            providers=[provider],
+            sess_options=session_options,
+            provider_options=None if provider_options is None else [provider_options],
```

Comment:
is `None` the default?

Reply:
Of `InferenceSession` in ONNX Runtime, yes: https://github.com/microsoft/onnxruntime/blob/94e34ace15f4c7011590d387c5dcabaa3e446d7f/onnxruntime/python/onnxruntime_inference_collection.py#L295